### PR TITLE
Adds operator and webhook v0.2.0-alpha01

### PR DESCRIPTION
**Pull request by cbron**

### CODE REVIEW

File: packages/rancher-operator-crd/package.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+url: https://github.com/rancher/rancher-operator/releases/download/v0.2.0-alpha01/rancher-operator-crd-0.2.0-alpha01.tgz
+packageVersion: 00
+releaseCandidateVersion: 08
```
Comment (aiyengar2) [inline packages/rancher-operator-crd/package.yaml:3]:
```suggestion
packageVersion: 01
releaseCandidateVersion: 00
```

File: packages/rancher-operator/package.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+url: https://github.com/rancher/rancher-operator/releases/download/v0.2.0-alpha01/rancher-operator-crd-0.2.0-alpha01.tgz
+packageVersion: 00
+releaseCandidateVersion: 08
```
Comment (aiyengar2) [inline packages/rancher-operator/package.yaml:3]:
```suggestion
packageVersion: 01
releaseCandidateVersion: 00
```

File: packages/rancher-webhook/package.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+url: https://github.com/rancher/webhook/releases/download/v0.2.0-alpha01/rancher-operator-crd-0.2.0-alpha01.tgz
```
Comment (aiyengar2) [inline packages/rancher-webhook/package.yaml:1]:
```suggestion
url: https://github.com/rancher/webhook/releases/download/v0.2.0-alpha01/rancher-webhook-0.2.0-alpha01.tgz
```

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

File: packages/rancher-operator-crd/package.yaml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+url: https://github.com/rancher/rancher-operator/releases/download/v0.2.0-alpha01/rancher-operator-crd-0.2.0-alpha01.tgz
```
Comment (aiyengar2) [inline packages/rancher-operator-crd/package.yaml:1]: FYI @cbron the version here will show up in the Staging branch as `0.2.0-alpha0101-rc00` and in the Live branch as `0.2.0-alpha0101`. Is that okay?

If not, you might need to patch the version for each of these packages by running `make prepare`, making the change, and then running `make patch`.

Same comment for the other two charts

**Review by aiyengar2 — APPROVED** [review]
LGTM, one minor clarification

File: packages/rancher-operator-crd/package.yaml (hunk 4 above)
Comment (cbron) [reply packages/rancher-operator-crd/package.yaml:1]: fixed. Its now `-alpha01-rc01` as we want `-alpha01` to be final release.

**Review by aiyengar2 — APPROVED** [review]

**Pull request merged**